README.md

```diff
@@ -1 +1,157 @@
+
+Please be familiar with conventional commit as described in the Contributing section below.
+
+Default preset used is angular for conventional commits, please see the [angular conventions](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum).
+
+The `<type>` `'breaking'` in the commit message will trigger a major version bump as well as any of the following text contained in the commit body: `"BREAKING CHANGE", "BREAKING CHANGES", "BREAKING_CHANGES", "BREAKING", "BREAKING_CHANGE"`. Please refer to the `.releaserc.json` file for the full configuration.
+
+The script `npm run release` will automatically trigger the release in CI. To manually test the release the following flags -`--dry-run --no-ci` - can be passed to the release script.
+
+Publishing and artifacts are managed separately by the pipeline.
+
+## Contributing
+
+To facilitate the standardisation of the code, a few helpers and tools have been adopted for this repository.
+
+### External dependencies
+
+The projects has multiple hooks configured using [husky](https://github.com/typicode/husky#readme) which will execute the following scripts: `audit`, `lint`, `build`, `test` and format your code with [eslint](https://github.com/typescript-eslint/typescript-eslint#readme) and [prettier](https://github.com/prettier/prettier).
+
+You will be required to install [git-secrets](https://github.com/awslabs/git-secrets) (_brew approach is recommended_) and DVSA [repo-security-scanner](https://github.com/UKHomeOffice/repo-security-scanner) that runs against your git log history to find accidentally committed passwords, private keys.
+
+We follow the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/) when we commit code to the repository and follow the [angular convention](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum).
+
+The type is mandatory and must be all lowercase.
+The scope of your commit remain is also mandator, it must include your ticket number and be all lowercase.
+
+```js
+// Please see /commitlint.config.js for customised format
+
+type(scope?): subject
+
+// examples
+'chore(cvsb-1234): my commit msg' // pass
+'CHORE(cvsb-1234): my commit msg' // will fail
+
+```
+
+### Code standards
+
+#### Code structure
+
+Domain Drive Design diagram with Interfaces, Application, Domain layers and Infrastructure across the layers.
+
+<p align="center">
+  <img src="./docs/DDD_architecture.jpg" alt="Domain Drive Design diagram with Interfaces, Application, Domain layers and Infrastructure across the layers" >
+</p>
+
+Further reading: [E. Evans in its Domain Driven Design approach](http://dddsample.sourceforge.net/architecture.html) and [The Clean Architecture](https://blog.cleancoder.com/uncle-bob/2012/08/13/the-clean-architecture.html).
```

Comment:
Not sure this is needed and there is an inclusivity issue here so best to remove - speak with me via teams in further explanation needed

Reply:
As discussed, I've removed this line.